content/blog/sprint-review-1916.md

```diff
@@ -0,0 +1,44 @@
+Title: Wazo Platform 19.16 Released
+Date: 2019-11-18
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: sprint-review-1916
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 19.16 release.
+
```

Comment:
Some projects include a small description of the project at the beginning of the release note.

Here's an example from glibc

```
The GNU C Library
=================

The GNU C Library version 2.28 is now available.

The GNU C Library is used as *the* C library in the GNU system and
in GNU/Linux systems, as well as many other systems that use Linux
as the kernel.

The GNU C Library is primarily designed to be a portable 
and high performance C library.  It follows all relevant 
standards including ISO C11 and POSIX.1-2008.  It is also 
internationalized and has one of the most complete 
internationalization interfaces known. 

The GNU C Library webpage is at http://www.gnu.org/software/libc/

Packages for the 2.28 release may be downloaded from:
        http://ftpmirror.gnu.org/libc/
        http://ftp.gnu.org/gnu/libc/

The mirror list is at http://www.gnu.org/order/ftp.html

NEWS for version 2.28
=====================

Major new features:
```

Are there any objections to include this description copied from our website?

```
Wazo Platform is an Open Source project allowing to build carrier grade programmable IP communication infrastructures. You can pick and choose the components you need to build your infrastructures with class 5 features like audio and video calls, chat, call centers, conferences, voicemail, etc., and class 4 features like security, routing, load balancing, etc. You can augment the platform and integrate it with your subscription and billing capabilities or any value-added services like Speech to Text, A.I. or sentiment analysis. These communication infrastructures can be deployed and scaled on bare metal, virtual machines or containers.
```

Reply:
I don't think it's really useful. Maybe in a changelog file when it can be redistributed without the context, but not in a blog post hosted by the company.
But my opinion is not strong, so I don't have real objection.